### ISSUE TITLE
"Plugin incompatible and requires its author to update"

### ISSUE THREAD

**Issue by SamCain3** [Adam-/runelite-plugins#80]

**Comment by griceyj** [comment]
Dead? :(

**Comment by CreativeTechGuy** [comment]
I assume this is about the [Bank Value Tracker](https://runelite.net/plugin-hub/show/bank-value-tracker) right?

**Comment by elwakka** [comment]
@CreativeTechGuy Yes.  For some reason the plugin page just links here.  Code is here: https://github.com/AdrianLeeElder/runelite-bank-history-plugin  The issue is it only works with usernames...  it doesn't work for jagex accounts.  Probably an easy fix.

**Comment by CreativeTechGuy** [comment]
@elwakka the plugin appeared to have been taken over by Adam a few years ago. This repo is actually the one that is live currently: https://github.com/runelite/plugin-hub/blob/master/plugins/bank-value-tracker
